stl/inc/system_error

```diff
@@ -432,8 +433,21 @@ protected:
 
 _CRTIMP2_PURE const char* __CLRCALL_PURE_OR_CDECL _Syserror_map(int);
 _CRTIMP2_PURE int __CLRCALL_PURE_OR_CDECL _Winerror_map(int);
-_CRTIMP2_PURE unsigned long __CLRCALL_PURE_OR_CDECL _Winerror_message(
-    unsigned long _Message_id, char* _Narrow, unsigned long _Size);
+
+struct _System_error_message {
+    char* _Str;
+    size_t _Length;
+
+    _System_error_message(unsigned long _Ec) noexcept

```

Comment:
`_Ec` could be `const`. We prefer value parameters to be `const` whenever possible in new code.